pkg/virt-controller/services/renderresources.go

```diff
@@ -212,14 +212,20 @@ func WithCPUPinning(cpu *v1.CPU) ResourceRendererOption {
 			}
 		}
 
-		// allocate 1 more pcpu if IsolateEmulatorThread request
+		// allocate pcpus for emulatorThread if IsolateEmulatorThread is requested
```

Comment:
```suggestion
		// allocate one or two pcpus for emulatorThread if IsolateEmulatorThread is requested
```

Reply:
I prefer to be generic. comment tend to get stale, keeping generally "plural" without being explicit will help keeping it true.